lession_13/homework_13.py

```diff
@@ -0,0 +1,24 @@
+
+class Rhombus:
+    def __init__(self, side_a, corner_a):
+        self.set_side_a(side_a)
```

Comment:
It is better to assign this attributes for at least for object first. 
For ex. self.side_a = __set_side_a(side_a) etc.